webpack/components/ResourceQuotaEmptyState/index.js

```diff
@@ -35,7 +35,7 @@ const ResourceQuotaEmptyState = () => {
         iconType="pf"
         header={__('Resource Quotas')}
         description={__(
-          'Resource Quotas help admins to manage hardware resources (like CPUs, RAM, and disk space) among users or usergroups. \n\rDefine a Resource Quota here and apply it to users in order to guarantee a free share of your resources.'
+          'Resource Quotas help admins to manage hardware resources including CPUs, memory, and disk space among users or usergroups. \n\rDefine a Resource Quota here and apply it to users in order to guarantee a free share of your resources.'
```

Comment:
After we have merged https://github.com/theforeman/foreman-documentation/pull/3460, we could also add a link to the official Foreman docs. Thorben and I have already implemented something similar in foreman_remote_execution. I believe there is a shared function that we can use which handles the logic so we'll only have to provide the name of the guide and the anchor.